modules/visual_script/visual_script.h

```diff
@@ -374,12 +375,19 @@ class VisualScriptInstance : public ScriptInstance {
 		int node;
 		int max_stack;
 		int trash_pos;
-		int return_pos;
```

Comment:
This `return_pos` was unused AFAICT, same for `valid`. If those were meant to be used somewhere, maybe that code was forgotten :)